src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -42,6 +42,11 @@ public class MsalException : Exception
         /// </summary>
         public const string BrokerErrorCode = "BrokerErrorCode";
 
+        /// <summary>
+        /// An <see cref="AdditionalExceptionData"/> property key, available when using desktop brokers.
```

Comment:
Does it mean that mobile brokers are excluded here?

Reply:
Yes.